src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let totalCost = days * 40;
```

Comment:
Prefer `const` over `let` for `totalCost` since its value is not being reassigned, which ensures the variable is not changed unintentionally elsewhere in the function.